inventory/README.md

```diff
@@ -2,45 +2,45 @@
   </a>
 </p>
 
-1. [Introduction](#introduction)
-2. [How it works](#how-it-works)
-3. [Implementation](#implementation)
-    1. [Microprofile](#microprofile)
-4. [Features and App details](#features)
-5. [Building the app](#building-the-app)
-6. [Setting up MYSQL](#setting-up-mysql)
-    1. [MYSQL Pre-requisites](#mysql-pre-requisites)
-    2. [Set Up MYSQL on Minikube](#set-up-mysql-on-minikube)
-    3. [Set Up MYSQL on IBM Cloud Private](#set-up-mysql-on-ibm-cloud-private)
-7. [Running the app and stopping it](#running-the-app-and-stopping-it)
-    1. [Pre-requisites](#pre-requisites)
-    2. [Locally in Minikube](#locally-in-minikube)
-    3. [Remotely in ICP](#remotely-in-icp)
-8. [References](#references)
-
-### Introduction
+## Table of Contents
+
+* [Introduction](#introduction)
+* [How it works](#how-it-works)
+* [Implementation](#implementation)
+    + [Microprofile](#microprofile)
+* [Features](#features)
+* [Deploying the App](#deploying-the-app)
+    + [IBM Cloud Private](#ibm-cloud-private)
+    + [Minikube](#minikube)
+* [Run Inventory Service locally](#run-inventory-service-locally)
+    + [Building the app](#building-the-app)
+    + [Setting up MySQL](#setting-up-mysql)
+    + [Setting up Zipkin](#setting-up-zipkin)
+    + [Running the app and stopping it](#running-the-app-and-stopping-it)
+* [References](#references)
+
+## Introduction
 
 This project demonstrates the implementation of Inventory Microservice. The inventory microservice uses MySQL database as its datasource. It has the complete list of items of our store front. 
 
 - Based on [MicroProfile](https://microprofile.io/).
 - Uses MySQL as the inventory database.
-- Deployment options for Minikube environment and ICP.
 
-### How it works
+## How it works
 
-Inventory Microservice serves 'IBM Cloud Native Reference Architecture' suite, available at https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes, Microservice-based reference application. Though it is a part of a bigger application, Inventory service is itself an application in turn that manages the data from data store. Catalog Microservice serves as the cache to the Inventory.
+Inventory Microservice serves 'IBM Cloud Native Reference Architecture' suite, available [here](https://github.com/ibm-cloud-architecture/refarch-cloudnative-kubernetes/tree/microprofile), Microservice-based reference application. Though it is a part of a bigger application, Inventory service is itself an application in turn that manages the data from data store. Catalog Microservice serves as the cache to the Inventory.
 
-### Implementation
+## Implementation
 
-#### [MicroProfile](https://microprofile.io/)
+### [MicroProfile](https://microprofile.io/)
 
 MicroProfile is an open platform that optimizes the Enterprise Java for microservices architecture. In this application, we are using [**MicroProfile 1.3**](https://github.com/eclipse/microprofile-bom). This includes
 
```

Comment:
In my opinion, we should simply omit the outdated features and leave what features are actually being used by their associated MicroProfile versions

Reply:
1.3 includes all the previous feature from 1.2 plus the additional ones. So just to get an overall view, I mentioned all of them there. Do you think we should just mention that we are on 1.3 now and remove the rest?

Reply:
Got you